src/catkin_pkg/package.py

```diff
@@ -370,6 +371,14 @@ def __str__(self):
         return txt
 
 
+# Subclassing ``str`` to keep backward compatibility.
+class License(str):
+    def __new__(cls, value, path=''):
+        obj = str.__new__(cls, str(value))
+        obj.path = path
```

Comment:
I would suggest to use `file` rather then `path` to follow the naming in XML.

Reply:
``file`` is a builtin function. But for this case, this will not cause a problem. I will use ``file`` instead of ``path`

Reply:
I think the class member should `file` (so `obj.file = ...`). But since `file` is a builtin in Python the argument of the function should be `file_` (so `file_=None`).

Reply:
OK. Fixed.